platform/wm/build/wm.rake

```diff
@@ -1590,7 +1590,7 @@ namespace "device" do
       Rake::Task['config:common:ymlsetup'].invoke
       container_path = determine_prebuild_path_win('wm', $app_config)
       $skip_build_extensions = true
-      $skip_build_js_api_files = true
+      $skip_build_js_api_files = false
```

Comment:
This was the reason api .js file was not getting generated after cleaning rhobundle/apps/api folder.